server/src/application/survey.ts

```diff
@@ -1479,14 +1488,24 @@ function dbSurveyToSurvey(
     },
     organization: dbSurvey.organization,
     tags: dbSurvey.tags,
+    enabledLanguages: dbSurvey.survey_languages,
```

Comment:
Should this column be renamed to `enabled_languages` to better match the source code? Might cause some confusion in the future... I guess even just `languages` would be fine, because the table name itself already indicates we're talking about surveys here.